deployments/warp_routes/WBTC/coti-ethereum-deploy.yaml

```diff
@@ -0,0 +1,14 @@
+coti:
+  decimals: 8
+  name: Wrapped BTC
+  owner: "0xdF2E2886d23ba57F996C203D2Ccd9dCa6373590C"
+  symbol: WBTC
+  type: synthetic
+ethereum:
+  coinGeckoId: wrapped-bitcoin
```

Comment:
should this be part of WarpCore?

Reply:
the coingecko id ? yes we have elsewhere too for the monitors

Reply:
i thought it was read from the WarpCore config. this is the deploy config

Reply:
for Eth, its in warpcore: https://github.com/hyperlane-xyz/hyperlane-registry/pull/847/files#diff-e0ef444fc2f76e96a9c12e2d9a76def7f2d44c62771c58788643734ee9212835R4

Reply:
ups reading warp core i thought this was warp core you are right it should be in the other file